contrib/helm/pganalyze-collector/README.md

```diff
@@ -46,6 +46,11 @@ pganalyze statistics collector
 | securityContext.runAsNonRoot | bool | `true` |  |
 | securityContext.runAsUser | int | `1000` |  |
 | securityContext.seccompProfile.type | string | `"RuntimeDefault"` |  |
+| service.create | bool | `false` | Specifies whether a service should be created for receiving logs via OpenTelemtry This service is used when Postgres is running within the cluster and Postgres logs are sent out to the collector using log collectors like Fluent Bit |
+| service.name | string | `"pganalyze-collector-otel-service"` | The name of the service to use. If not set and create is true, a name is generated using the fullname template. This is the name referenced by the log sender like Fluent Bit |
+| service.port | int | `4318` | The port of service. This is the port referenced by the log sender like Fluent Bit |
+| service.targetPort | int | `4318` | The target port of the log OTEL server port. This should match to the port number specified with db_log_otel_server |
+| service.type | string | `"ClusterIP"` | The type of service to create. |
```

Comment:
Yeah, it's k8s terminology. In k8s, a Service is an abstraction which defines a logical set of Pods and a policy by which to access them, often over a network.
Services in k8s support different types, primarily: ClusterIP (default, providing internal cluster connectivity), NodePort (exposes the Service on each Node’s IP at a static port), and LoadBalancer (provides a fixed, external IP assigned by the cloud provider and routes traffic to the NodePort or ClusterIP).
Here, I'm putting "ClusterIP" as a sample/default value and it'll work when you put the Postgres, fluent-bit, and the collector in the same k8s cluster (and this helm chart is mainly assuming that it'll be the case, otherwise they'll need to make more changes).